assets/js/blocks/attribute-filter/frontend.ts

```diff
@@ -26,8 +26,8 @@ const getProps = ( el: HTMLElement ) => {
 				el.dataset.displayStyle ||
 				metadata.attributes.displayStyle.default,
 			showFilterButton: el.dataset.showFilterButton === 'true',
-			isPreview: false,
-			className: el.dataset.className || '',
```

Comment:
Is the removal of `isPreview` and `className` intentional?

Reply:
> Is the removal of `isPreview` and `className` intentional?

@Aljullu Yes, I removed them because we don't need them on the front end.
- `isPreview` is only used in the block inserter.
- `className` is handled by the `<Save>` component, we don't use the `className` attribute in the `<Block>` as well.